test/NuGet.Tests.Apex/NuGet.Tests.Apex/Apex/ApexTestContext.cs

```diff
@@ -22,7 +22,7 @@ internal class ApexTestContext : IDisposable
 
         public NuGetApexTestService NuGetApexTestService { get; }
 
-        public ApexTestContext(VisualStudioHost visualStudio, ProjectTemplate projectTemplate, ILogger logger, bool noAutoRestore = false)
+        public ApexTestContext(VisualStudioHost visualStudio, ProjectTemplate projectTemplate, ILogger logger, bool noAutoRestore = false, bool addNetStandardFeeds = false)
```

Comment:
From what I can tell, there are only 5 places where `ApexTestContext` is created, and you're setting `addNetStandardFeeds` to true in all of them. In other words, this parameter does not appear useful as it's always true.

Reply:
@zivkan Thanks for your comments.
While I searched and found 20+  `ApexTestContext` . Those 5 create netstandard2.0 project so they need the `addNetStandardFeeds ` to be true.

Reply:
right, I was mislead by [GitHub's code search results](https://github.com/NuGet/NuGet.Client/search?q=ApexTestContext&unscoped_q=ApexTestContext).

Still, I'm not sure there's any harm to always adding the nuget.client\packages folder as an additional source, so I don't know if the additional complexity of the new argument is worthwhile. But it's not important.